packages/venia-concept/src/components/MiniCart/miniCart.js

```diff
@@ -31,10 +33,26 @@ class MiniCart extends Component {
         data: mockData
     };
 
+    async componentDidMount() {
+        const {
```

Comment:
This async load pattern is intriguing; do you intend for this to become a pattern that we'd repeat in other components, or even abstract away?

Reply:
Actually, it's just sort of a convenient break point. I know that `MiniCart` will be mounted in an inactive/hidden state, so I can request some of its children asynchronously and have them rendered before the user activates and sees the component.

We're going to need a more sophisticated solution, probably involving redux actions, to really do this right, I think.